contracts/proxy/Clones.sol

```diff
@@ -23,6 +23,7 @@ library Clones {
      * This function uses the create opcode, which should never revert.
      */
     function clone(address implementation) internal returns (address instance) {
+        /// @solidity memory-safe-assembly
         assembly {
             let ptr := mload(0x40)
```

Comment:
since this is using memory allocated at the free pointer, I think it [should be safe](https://docs.soliditylang.org/en/v0.8.13/assembly.html#memory-safety).

Reply:
Exactly!